lib/perl/Genome/Model/Tools/Speedseq/Sv.t

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env genome-perl
+
+use strict;
+use warnings;
+
+use above 'Genome';
+use Test::More;
+use Genome::File::Vcf::Differ;
+use Genome::Utility::Test qw(compare_ok);
+use Genome::Test::Data qw(get_test_file);
+
+my $pkg = 'Genome::Model::Tools::Speedseq::Sv';
+use_ok($pkg);
+
+my $speedseq_version = '0.0.3a-gms';
+
+my $expected_output_dir = __FILE__.".out";
+
+my $reference_fasta = get_test_file('NA12878', 'human_g1k_v37_20_42220611-42542245.fasta');
+my $bam = get_test_file('NA12878', 'NA12878.20slice.30X.aligned.bam');
+my $split_bam = get_test_file('NA12878','NA12878.20slice.30X.splitters.bam');
```

Comment:
let's use the 30X.aligned.* versions of these files and remove the other ones that you added - they are identical except for the file name.